### PR TITLE
Implement Admin & Superadmin UI with Org switching

### PR DESCRIPTION
**Pull request by jaredscheib**
- [x] Rebased/mergable
  - [x] Tests pass

Connect #1829
Connect #1896    
Connect #1917
Connect #1926
Connect #1938  
Connect #1830 
Connect #2199 - redirect user where appropriate based on auth when logging in
Connect #2168 – org switching nav block addressed by 9d68ecf
Connect #2259
Connect #2318 
Connect #2310 

Current design:
[chronograf-superadmin-panel--v2.pdf](https://github.com/influxdata/chronograf/files/1423782/chronograf-superadmin-panel--v2.pdf)

### WIP TODO
1. [x] clear filter to get back to all users
1. [x] make checkboxen look nice
1. [x] make selection retention work more clearly
1. [x] show number of users selected
1. [x] show batch actions
1. [x] dropdown for role per user
1. [x] dropdown for org per user
1. [x] prevent adding role until part of an organization
1. [x] create organization
1. [x] read superadmin status from bool on /me response
1. [x] refactor chronograf/UsersTable into separate Orgs & Users views
1. [x] implement admin vs superadmin view with Authorized component
  - [x] create org button
  - [x] filter by org dropdown
  - [x] show only Org view
  - [x] hide superadmin column
1. [x] rename & delete organization
1. [x] dropdown to make user a superadmin
1. [x] add individual user actions (delete, etc.)
1. [x] create a user (whitelist for org)
1. [x] wire up
1. [x] hide chronograf user admin menu item when not using auth
1. [x] remove batch actions
1. [x] add back in delete for individual user

### set up an OAuth2 application for your local Chronograf instance, ex. on GitHub:
1. visit https://github.com/settings/developers
1. click `New OAuth App` app
1. Application name: anything, e.g. `Chronograf`
1. Homepage URL: `http://localhost:8888`
1. Authorization callback URL: `http://localhost:8888/oauth/github/callback`

### build the right server
1. `git checkout origin/multitenancy_default_org`
1. `make run-dev` from there
1. kill the server and run again with `./chronograf -d -i <your-client-id> -s <your-auth-secret> -t supersecret2 --auth-duration=60m --custom-link=InfluxData:https://influxdata.com --custom-link=Pineapple:http://reddit.com/r/KnightsOfPineapple/`

### build the right client
1. with that server running, `git checkout origin/multitenancy_ui_superadmin_admin_panel`
1. `yarn start` on that branch
1. with auth now turned on, explore the app
1. if you want to see what it's like to use the app as other role types (since you will be a superadmin if you're the only user and running it locally), at least as a client-side hack you can change `role: getMeRole(me),` to `role: 'admin' || 'editor' || 'viewer' || 'member' || getMeRole(me),` in `ui/src/shared/reducers/auth.js`, save it, let yarn rebuild the client, and explore the app, and then when you want to see it as an Editor, lop off 'admin ||', save, let yarn rebuild, etc.

### once you've thoroughly tested it there:
1. kill the server process
1. check out the server branch again
1. run the server simply with `/.chronograf -d`
1. check out the client branch again
1. let `yarn start` do its thing, or make it run if you killed it
1. with auth now turned off, explore the app

### CODE REVIEW

File: ui/src/admin/components/chronograf/OrgTableRow.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      </td>
+      <td style={{width: colOrg}}>
+        <span className="chronograf-user--org">
+          {organization.name}
+        </span>
+      </td>
+      <td style={{width: colRole}}>
+        <span className="chronograf-user--role">
+          <Dropdown
+            items={USER_ROLES.map(r => ({
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/OrgTableRow.js]: This should be performed alongside the other variable declarations above

File: ui/src/admin/components/chronograf/Organization.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    this.setState({isDeleting: true})
+  }
+
+  handleDismissDeleteConfirmation = () => {
+    this.setState({isDeleting: false})
+  }
+
+  handleDeleteOrg = () => {
+    const {onDelete, organization} = this.props
+    this.setState({isDeleting: false})
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/Organization.js]: Do you need this if the DOM node is going away?

File: ui/src/admin/components/chronograf/UsersTable.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                        isSameUser={isSameUser}
+                        organization={organization}
+                        onSelectAddUserToOrg={this.handleSelectAddUserToOrg(
+                          user
+                        )}
+                        onChangeUserRole={this.handleChangeUserRole}
+                      />
+              )
+            : <tr className="table-empty-state">
+                <Authorized
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTable.js]: This looks like it's only controlling the width of the resulting `th`

File: ui/src/admin/components/chronograf/UsersTable.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+import Authorized, {SUPERADMIN_ROLE} from 'src/auth/Authorized'
+
+import UsersTableRow from 'src/admin/components/chronograf/UsersTableRow'
+import OrgTableRow from 'src/admin/components/chronograf/OrgTableRow'
+
+import {DEFAULT_ORG_NAME} from 'src/admin/constants/dummyUsers'
+import {USERS_TABLE} from 'src/admin/constants/chronografTableSizing'
+
+class ChronografUsersTable extends Component {
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTable.js]: This doesn't match the name of the file

File: ui/src/admin/components/chronograf/UsersTableHeader.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import React, {Component, PropTypes} from 'react'
+
+import Authorized, {SUPERADMIN_ROLE} from 'src/auth/Authorized'
+
+import Dropdown from 'shared/components/Dropdown'
+
+class UsersTableHeader extends Component {
+  constructor(props) {
+    super(props)
+  }
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTableHeader.js]: Is this doing anything?

File: ui/src/admin/components/chronograf/UsersTableOrgCell.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        />
+      </td>
+    )
+  }
+
+  return organizations.length
+    ? <td style={{width: colOrg}}>
+        {user.roles
+          .filter(role => {
+            return !(role.organization === DEFAULT_ORG_ID)
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTableOrgCell.js]: `role.organization !== DEFAULT_ORG_ID`

File: ui/src/admin/components/chronograf/UsersTableRoleCell.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+import Dropdown from 'shared/components/Dropdown'
+
+import {DEFAULT_ORG_ID, USER_ROLES} from 'src/admin/constants/dummyUsers'
+import {USERS_TABLE} from 'src/admin/constants/chronografTableSizing'
+
+const UsersTableRoleCell = ({user, onChangeUserRole}) => {
+  const {colRole} = USERS_TABLE
+
+  // User must be part of more than one organization to be able to be assigned a role
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTableRoleCell.js]: How come?

File: ui/src/admin/components/chronograf/UsersTableRoleCell.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+        <span className="chronograf-user--role">N/A</span>
+      </td>
+    )
+  }
+
+  return (
+    <td style={{width: colRole}}>
+      {user.roles
+        .filter(role => {
+          return !(role.organization === DEFAULT_ORG_ID)
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTableRoleCell.js]: !==

File: ui/src/admin/components/chronograf/UsersTableRow.js [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+  selectedUsers,
+  isSameUser,
+  onSelectAddUserToOrg,
+  onChangeUserRole,
+  onChooseFilter,
+  onChangeSuperAdmin,
+}) => {
+  const {colProvider, colScheme} = USERS_TABLE
+
+  const isSelected = selectedUsers.find(u => isSameUser(user, u))
```
Comment (lukevmorris) [inline ui/src/admin/components/chronograf/UsersTableRow.js]: some?

File: ui/src/admin/containers/AdminChronografPage.js [hunk 10]
```diff
@@ -0,0 +1,383 @@
...
+  NO_ORG,
+  USER_ROLES,
+} from 'src/admin/constants/dummyUsers'
+
+class AdminChronografPage extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      // TODO: pass around organization object instead of just name
```
Comment (lukevmorris) [inline ui/src/admin/containers/AdminChronografPage.js]: Is this done?

File: ui/src/admin/containers/AdminChronografPage.js [hunk 11]
```diff
@@ -0,0 +1,383 @@
...
+
+    if (
+      user.roles.length === 1 &&
+      user.roles[0].organization !== DEFAULT_ORG_ID
+    ) {
+      newUser = {
+        ...newUser,
+        roles: [
+          ...newUser.roles,
+          {organization: DEFAULT_ORG_ID, name: MEMBER_ROLE},
```
Comment (lukevmorris) [inline ui/src/admin/containers/AdminChronografPage.js]: You may want to add a comment explaining why you're doing this

File: ui/src/admin/containers/AdminChronografPage.js [hunk 12]
```diff
@@ -0,0 +1,383 @@
...
+          organization: organization.id,
+        },
+      ],
+    })
+  }
+  handleRemoveUserFromOrg = (user, organization) => {
+    const {actions: {updateUserAsync}} = this.props
+
+    let newRoles = user.roles.filter(r => r.organization !== organization.id)
+    if (newRoles.length === 0) {
```
Comment (lukevmorris) [inline ui/src/admin/containers/AdminChronografPage.js]: How would this happen? Can a user be removed from the Default Org?

File: ui/src/admin/containers/AdminChronografPage.js [hunk 13]
```diff
@@ -0,0 +1,383 @@
...
+                        onAddUsersToOrg={this.handleBatchAddUsersToOrg}
+                        onRemoveUsersFromOrg={
+                          this.handleBatchRemoveUsersFromOrg
+                        }
+                        onChangeRoles={this.handleBatchChangeUsersRole}
+                      />
+                      <div className="panel-body chronograf-admin-table--panel">
+                        <Authorized
+                          requiredRole={SUPERADMIN_ROLE}
+                          propsOverride={{organizationName: organization.name}}
```
Comment (lukevmorris) [inline ui/src/admin/containers/AdminChronografPage.js]: Does `UsersTable` accept `organizationName`?

File: ui/src/admin/containers/AdminChronografPage.js [hunk 14]
```diff
@@ -0,0 +1,383 @@
...
+                            organizations={organizations}
+                            onFilterUsers={this.handleFilterUsers}
+                            onToggleUserSelected={this.handleToggleUserSelected}
+                            selectedUsers={selectedUsers}
+                            isSameUser={isSameUser}
+                            onToggleAllUsersSelected={
+                              this.handleToggleAllUsersSelected
+                            }
+                            onAddUserToOrg={this.handleAddUserToOrg}
+                            onUpdateUserRole={this.handleUpdateUserRole()}
```
Comment (lukevmorris) [inline ui/src/admin/containers/AdminChronografPage.js]: Why are these wrapped in a function?

File: ui/src/auth/Authorized.js [hunk 15]
```diff
@@ -51,10 +50,12 @@ const Authorized = ({
...
 
-  const meRole = getMeRole(me)
+  if (!isUsingAuth) {
+    return replaceWithWhenNotUsingAuth || firstChild
+  }
 
-  if (!isUsingAuth || isUserAuthorized(meRole, requiredRole)) {
-    return firstChild
+  if (isUserAuthorized(meRole, requiredRole)) {
+    return replaceWith || firstChild
```
Comment (lukevmorris) [inline ui/src/auth/Authorized.js]: We discussed in Slack that this degrades the clarity of the HOC's functionality

File: ui/src/side_nav/components/UserNavBlock.js [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+import {bindActionCreators} from 'redux'
+
+import classnames from 'classnames'
+
+import {meChangeOrganizationAsync} from 'shared/actions/auth'
+
+class UserNavBlock extends Component {
+  constructor(props) {
+    super(props)
+  }
```
Comment (lukevmorris) [inline ui/src/side_nav/components/UserNavBlock.js]: Does this do anything?

File: ui/src/side_nav/components/UserNavBlock.js [hunk 17]
```diff
@@ -0,0 +1,123 @@
...
+              >
+                {organizations.find(o => o.id === role.organization).name}{' '}
+                <strong>({role.name})</strong>
+              </span>
+            )
+          })}
+          <a className="sidebar-menu--item" href={logoutLink}>
+            Logout
+          </a>
+          {customLinks ? <div className="sidebar-menu--divider" /> : null}
```
Comment (lukevmorris) [inline ui/src/side_nav/components/UserNavBlock.js]: Can't the truthy `customLinks` behavior be consolidated into a single block?

File: ui/src/auth/Authorized.js (hunk 15 above)
Comment (jaredscheib) [reply ui/src/auth/Authorized.js]: conclusions:

Comment (jaredscheib) [reply ui/src/auth/Authorized.js]: takeaways: 1. replace `replaceWith` with `replaceWithIfAuthorized` and `replaceWithIfNotAuthorized` and amend the above logic accordingly. then, correct ```<Authorized
                requiredRole={SUPERADMIN_ROLE}
                replaceWith={
                  <input
                    type="text"
                    value={userOrganization.name}
                    disabled={true}
                    className="form-control input-sm disabled"
                  />
                }
              >``` to use that new API and replace all other instances of `replaceWith` with the correct new API

File: ui/spec/admin/reducers/chronografSpec.js [hunk 18]
```diff
@@ -0,0 +1,176 @@
...
+    provider: 'Google',
+    scheme: 'OAuth2',
+    superadmin: true,
+    roles: [
+      {organizationName: 'All Users', organizationID: 666, name: NO_ROLE},
+      {organizationName: 'Blue Team', organizationID: 1234, name: NO_ROLE},
+    ],
+    links: {self: '/chronograf/v1/users/47119'},
+  },
+]
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/spec/admin/reducers/chronografSpec.js]: They were previously dummy data for the development of the UI

File: ui/src/admin/apis/chronograf.js [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+      url: updatedUserWithRolesOnly.links.self,
+      data: updatedUserWithRolesOnly,
+    })
+  } catch (error) {
+    console.error(error)
+    throw error
+  }
+}
+
+export const deleteUser = async url => {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/apis/chronograf.js]: good catch

[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/apis/chronograf.js]: good call

File: ui/src/admin/apis/chronograf.js [hunk 20]
```diff
@@ -0,0 +1,104 @@
...
+      url,
+      data: organization,
+    })
+  } catch (error) {
+    console.error(error)
+    throw error
+  }
+}
+
+export const renameOrganization = async organization => {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/apis/chronograf.js]: all you can do an organization currently is rename it, but i can change the name and possibly comment that

File: ui/src/admin/components/chronograf/BatchActionsBar.js [hunk 21]
```diff
@@ -0,0 +1,86 @@
+import React, {PropTypes} from 'react'
+
+import Dropdown from 'shared/components/Dropdown'
+
+import {DEFAULT_ORG, NO_ORG, USER_ROLES} from 'src/admin/constants/dummyUsers'
+
+const BatchActionsBar = ({
+  onDeleteUsers,
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/BatchActionsBar.js]: got rid of batch actions for now

File: ui/src/admin/components/chronograf/BatchActionsBar.js [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+  onAddUsersToOrg,
+  onRemoveUsersFromOrg,
+  onChangeRoles,
+  numUsersSelected,
+  organizationName,
+  organizations,
+}) => {
+  const showWhenAllUsers = organizationName === DEFAULT_ORG_NAME
+  const showWhenNotAllUser = organizationName !== DEFAULT_ORG_NAME
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/BatchActionsBar.js]: got rid of batch actions & selecting users for now

File: ui/src/admin/components/chronograf/BatchActionsBar.js [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+                  items={sanitizedOrgs}
+                  selected={'Remove from Organization'}
+                  onChoose={onRemoveOrgs}
+                  buttonColor="btn-primary"
+                  className="dropdown-200"
+                />
+              : null}
+            {showWhenNotAllUser
+              ? <Dropdown
+                  items={USER_ROLES.map(role => ({
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/BatchActionsBar.js]: 👍 but got rid of batch actions for now

File: ui/src/admin/components/chronograf/CreateUserOverlay.js [hunk 24]
```diff
@@ -0,0 +1,191 @@
...
+
+import Dropdown from 'shared/components/Dropdown'
+
+import {SUPERADMIN_OPTION_ITEMS} from 'src/admin/constants/dummyUsers'
+
+class CreateUserOverlay extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/CreateUserOverlay.js]: fixed

File: ui/src/admin/components/chronograf/CreateUserOverlay.js [hunk 25]
```diff
@@ -0,0 +1,191 @@
...
+    const {
+      userName,
+      userProvider,
+      userScheme,
+      userRole,
+      userSuperAdmin,
+      userOrganization,
+    } = this.state
+
+    const allowCreate =
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/CreateUserOverlay.js]: 👍

File: ui/src/admin/components/chronograf/CreateUserOverlay.js [hunk 26]
```diff
@@ -0,0 +1,191 @@
...
+          organization: userOrganization.id,
+        },
+      ],
+    }
+
+    onCreateUser(newUser)
+    onDismiss()
+  }
+
+  handleInputFocus = e => {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/CreateUserOverlay.js]: 👍

File: ui/src/admin/components/chronograf/NewOrganization.js [hunk 27]
```diff
@@ -0,0 +1,86 @@
+import React, {Component, PropTypes} from 'react'
+
+class Organization extends Component {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/NewOrganization.js]: yop - good catch

File: ui/src/admin/components/chronograf/NewOrganization.js [hunk 28]
```diff
@@ -0,0 +1,86 @@
...
+      organizationName: 'Untitled Organization',
+    }
+  }
+
+  handleKeyDown = e => {
+    const {onCancelCreateOrganization} = this.props
+
+    if (e.key === 'Escape') {
+      onCancelCreateOrganization()
+      this.inputRef.blur()
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/NewOrganization.js]: good call

File: ui/src/admin/components/chronograf/NewOrganization.js [hunk 29]
```diff
@@ -0,0 +1,86 @@
...
+  handleInputFocus = e => {
+    e.target.select()
+  }
+
+  handleClickSave = () => {
+    const {onCancelCreateOrganization, onCreateOrganization} = this.props
+    const {organizationName} = this.state
+
+    onCreateOrganization(organizationName)
+    onCancelCreateOrganization()
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/NewOrganization.js]: it was just being used to hide the component by calling setState false -- replaced it with a direct setState where necessary and removed this confusion

File: ui/src/admin/components/chronograf/OrgTableRow.js [hunk 30]
```diff
@@ -0,0 +1,92 @@
...
+  user,
+  organization,
+  onToggleUserSelected,
+  selectedUsers,
+  isSameUser,
+  onChangeUserRole,
+}) => {
+  const {colOrg, colRole, colSuperAdmin, colProvider, colScheme} = USERS_TABLE
+
+  const isSelected = selectedUsers.find(u => isSameUser(user, u))
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/OrgTableRow.js]: removed selectability from the entire scheme

File: ui/src/admin/components/chronograf/OrgTableRow.js (hunk 1 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/OrgTableRow.js]: 👍

File: ui/src/admin/components/chronograf/Organization.js [hunk 31]
```diff
@@ -0,0 +1,134 @@
...
+  }
+
+  handleNameClick = () => {
+    this.setState({isEditing: true})
+  }
+
+  handleInputBlur = reset => e => {
+    const {onRename, organization} = this.props
+
+    if (!reset && organization.name !== e.target.value) {
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/Organization.js]: 👍

File: ui/src/admin/components/chronograf/Organization.js [hunk 32]
```diff
@@ -0,0 +1,134 @@
...
+  }
+
+  handleKeyDown = e => {
+    if (e.key === 'Enter') {
+      this.inputRef.blur()
+    }
+    if (e.key === 'Escape') {
+      this.setState(
+        {reset: true, workingName: this.props.organization.name},
+        () => this.inputRef.blur()
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/Organization.js]: 👍

File: ui/src/admin/components/chronograf/Organization.js (hunk 2 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/Organization.js]: 👍

File: ui/src/admin/components/chronograf/Organization.js [hunk 33]
```diff
@@ -0,0 +1,134 @@
...
+              placeholder="Name this Organization..."
+              autoFocus={true}
+              onFocus={this.handleFocus}
+              ref={r => (this.inputRef = r)}
+            />
+          : <div
+              className="manage-orgs-form--name"
+              onClick={this.handleNameClick}
+            >
+              {workingName}
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/Organization.js]: now using workingName to track ongoing state, so this is desired

File: ui/src/admin/components/chronograf/Organization.js [hunk 34]
```diff
@@ -0,0 +1,134 @@
...
+            </div>}
+        {isDeleting
+          ? <div className="btn btn-sm btn-default btn-square manage-orgs-form--delete active">
+              <span className="icon trash" />
+              <ConfirmDeleteOrg
+                onDelete={this.handleDeleteOrg}
+                handleClickOutside={this.handleDismissDeleteConfirmation}
+              />
+            </div>
+          : <button
```
[the remark this answers is not in the document]
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/Organization.js]: i don't follow

File: ui/src/admin/components/chronograf/UsersTable.js (hunk 4 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTable.js]: 👍

File: ui/src/admin/components/chronograf/UsersTableHeader.js (hunk 5 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTableHeader.js]: updated. now has state

File: ui/src/admin/components/chronograf/UsersTable.js (hunk 3 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTable.js]: correct -- superadmin th is also hidden, and this goes along with it

File: ui/src/admin/components/chronograf/UsersTableOrgCell.js (hunk 6 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTableOrgCell.js]: no longer filtering

File: ui/src/admin/components/chronograf/UsersTableRoleCell.js (hunk 8 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTableRoleCell.js]: no longer a component -- simplified chronograf admin

File: ui/src/admin/components/chronograf/UsersTableRow.js (hunk 9 above)
Comment (jaredscheib) [reply ui/src/admin/components/chronograf/UsersTableRow.js]: no longer allowing selection for now

File: ui/src/admin/containers/AdminChronografPage.js (hunk 10 above)
Comment (jaredscheib) [reply ui/src/admin/containers/AdminChronografPage.js]: 👍

File: ui/src/admin/containers/AdminChronografPage.js (hunk 11 above)
Comment (jaredscheib) [reply ui/src/admin/containers/AdminChronografPage.js]: removed that logic as it's for an obsolete approach

File: ui/src/admin/containers/AdminChronografPage.js (hunk 12 above)
Comment (jaredscheib) [reply ui/src/admin/containers/AdminChronografPage.js]: yes they can, but this code looks obsolete

File: ui/src/admin/containers/AdminChronografPage.js (hunk 13 above)
Comment (jaredscheib) [reply ui/src/admin/containers/AdminChronografPage.js]: no longer filtering, so obsolete

File: ui/src/admin/containers/AdminChronografPage.js (hunk 14 above)
Comment (jaredscheib) [reply ui/src/admin/containers/AdminChronografPage.js]: outdated

File: ui/src/auth/Authorized.js (hunk 15 above)
Comment (jaredscheib) [reply ui/src/auth/Authorized.js]: fixed -- great catch

File: ui/src/side_nav/components/UserNavBlock.js (hunk 16 above)
Comment (jaredscheib) [reply ui/src/side_nav/components/UserNavBlock.js]: nope, good catch

**Pull request merged**